### PR TITLE
Obfuscate freq visualization

### LINKED ISSUE #30: Regroup Frequency Visualizations
Network visualizations currently showing full IP addresses. Need to obfuscate those numbers and classes without showing specific data (see #33).

### LINKED ISSUE #32: Create PCAP loader to retrieve data based on specific filenames
- Retrieve specific data from datastore 
- Send to visualizations

### PR DESCRIPTION
**Pull request by mnuzen**
# Description

- Added PacketLoaderServlet to retrieve specific datastore entities based on user input
- Obfuscating frequency visualizations to depict non-descriptive groups and addresses

# Checklist:

- [x] Code follows the style guidelines
- [x] Code commented in relevant places
- [x] Changes to necessary documentation made
- [x] New and existing unit tests pass locally with changes

# Tests Performed:

- [x] Upload data to Datastore
- [x] Retreive data from Datastore
- [x] Map Visualization 
- [x] Network Map Visualization 
- [x] List other manual test cases performed

### CODE REVIEW

**Comment by mnuzen** [comment]
- created basic visualizations of nodes without any descriptive labels
- added functionality to depict certain files based on user input

Screenshot 2020-08-07 at 11 47 01 AM

Screenshot 2020-08-07 at 11 45 45 AM

File: src/main/java/com/google/netpcapanalysis/servlets/PacketLoaderServlet.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String json = convertToJsonUsingGson(data.getPCAPObjects(FILENAME));
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    FILENAME = getParameter(request, "file-input", "");
+    response.sendRedirect("/network.html"); // is there a way to put down the website that sent the request?
```
Comment (arunk054) [inline src/main/java/com/google/netpcapanalysis/servlets/PacketLoaderServlet.java:68]: Can you elaborate what is this question about?

File: src/main/java/com/google/netpcapanalysis/servlets/PacketLoaderServlet.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String json = convertToJsonUsingGson(data.getPCAPObjects(FILENAME));
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    FILENAME = getParameter(request, "file-input", "");
```
Comment (arunk054) [inline src/main/java/com/google/netpcapanalysis/servlets/PacketLoaderServlet.java:67]: should this be in the doGet()?

**Review by arunk054 — APPROVED** [review]
This is good. Let not worry too much about hiding the IPs. The main goal is to show that our design accommodates obfuscating the IPs.

**Review by Erik-Handeland — APPROVED** [review]

**Pull request merged**